### PR TITLE
Don't initialize tracer until device is unlocked.

### PR DESCRIPTION
**Pull request by vkryachko**

### CODE REVIEW

File: firebase-datatransport/src/main/java/com/google/firebase/datatransport/TransportRegistrar.java [hunk 1]
```diff
@@ -22,21 +22,23 @@
...
         Component.builder(TransportFactory.class)
             .add(Dependency.required(Context.class))
             .factory(
                 c -> {
                   TransportRuntime.initialize(c.get(Context.class));
                   return TransportRuntime.getInstance().newFactory(CCTDestination.LEGACY_INSTANCE);
                 })
-            .build());
+            .build(),
+        LibraryVersionComponent.create("fire-transport", BuildConfig.VERSION_NAME));
```
Comment (yifanyang) [inline firebase-datatransport/src/main/java/com/google/firebase/datatransport/TransportRegistrar.java:42]: Is this related to this pull request?

**Review by yifanyang — APPROVED** [review]

File: firebase-datatransport/src/main/java/com/google/firebase/datatransport/TransportRegistrar.java (hunk 1 above)
Comment (vkryachko) [reply firebase-datatransport/src/main/java/com/google/firebase/datatransport/TransportRegistrar.java:42]: Not exactly, but it makes us to measure how long firelog takes to start, will keep it for now.
